articles/api/authentication/_logout.md

```diff
@@ -24,50 +32,62 @@ curl --request GET \
   });
   
   webAuth.logout({
-    returnTo: 'YOUR_LOGOUT_URL',
-    client_id: '${account.clientId}'
+    post_logout_redirect_uri: 'YOUR_LOGOUT_URL',
+    id_token_hint: 'YOUR_ID_TOKEN_HINT',
+    logout_hint: 'YOUR_LOGOUT_HINT'
   });
 </script>
 ```
 
 <%= include('../../_includes/_http-method', {
   "http_badge": "badge-primary",
   "http_method": "GET",
-  "path": "/v2/logout",
+  "path": "/oidc/logout",
   "link": "#logout"
 }) %>
 
-Use this endpoint to logout a user. If you want to navigate the user to a specific URL after the logout, set that URL at the `returnTo` parameter. The URL should be included in any the appropriate `Allowed Logout URLs` list:
-- If the `client_id` parameter is included, the `returnTo` URL must be listed in the `Allowed Logout URLs` set at the application level (see [Setting Allowed Logout URLs at the App Level](/logout#set-the-allowed-logout-urls-at-the-application-level)).
-- If the `client_id` parameter is NOT included, the `returnTo` URL must be listed in the `Allowed Logout URLs` set at the tenant level (see [Setting Allowed Logout URLs at the Tenant Level](/logout#set-the-allowed-logout-urls-at-the-tenant-level)).
-- If the `client_id` parameter is included and the `returnTo` URL is NOT set, the server returns the user to the first Allowed Logout URLs set in the Dashboard (see [Setting Allowed Logout URLs at the Tenant Level](/logout#set-the-allowed-logout-urls-at-the-tenant-level)).
+Use this endpoint to logout a user. If you want to navigate the user to a specific URL after the logout, set that URL at the `post_logout_redirect_uri` parameter. The URL should be included in any the appropriate `Allowed Logout URLs` list:
```

Comment:
In the sentence: "The URL should be included in any" is a little awkward. Should it be: "The URL should be included in the appropriate `Allowed Logout URLs` list" or "The URL should be included in any of the the appropriate `Allowed Logout URLs` lists"?